proto/regen/ecocredit/v1/state.proto

```diff
@@ -83,6 +83,7 @@ message Project {
     index : {id : 2, fields : "class_key,id", unique : true}
     index : {id : 3, fields : "admin"}
     index : {id : 4, fields : "reference_id"}
+    index : {id : 5, fields : "class_key,reference_id"}
```

Comment:
that brings us back to here https://github.com/regen-network/regen-ledger/pull/1174#discussion_r897059786, what to do when no ref id is provided?

Reply:
Right... I'm seeing the comment above now about multiple projects with the same reference id and deciding to pick the first project rather than error.